src/modules/endpoint/__tests__/DefaultDataContainer.spec.js

```diff
@@ -0,0 +1,41 @@
+// @flow
+
+import DefaultDataContainer from '../DefaultDataContainer'
+import { CityModel } from '@integreat-app/integreat-api-client'
+import DatabaseContext from '../DatabaseContext'
+import RNFetchBlob from 'rn-fetch-blob'
+
+jest.mock('rn-fetch-blob')
+
+const testCity = new CityModel({
+  name: 'testCityName',
+  code: 'tcc',
+  live: true,
+  eventsEnabled: true,
+  extrasEnabled: true,
+  sortingName: 'testCity',
+  prefix: 'Stadt'
+})
+
+describe('DefaultDataContainer', () => {
```

Comment:
Tests for a lot of functions are missing here.

A test which is needed here, should check whether the loading for different cities/languages works.
This means one should be able to get the languages for a city A after getting them for city B.